### PR TITLE
Import Unicode properties

### PR DESCRIPTION
**Pull request by camertron**
This PR adds a bunch of importers to handle the many different files that make up the UCD's total set of properties.

--
This PR is part of a larger effort to refactor our Unicode properties support. See #170.

### CODE REVIEW

File: lib/twitter_cldr/resources/download.rb [hunk 1]
```diff
@@ -6,8 +6,13 @@
 module TwitterCldr
   module Resources
 
-    CLDR_URL = 'http://unicode.org/Public/cldr/26/core.zip'
-    ICU4J_URL = 'http://download.icu-project.org/files/icu4j/54.1/icu4j-54_1.jar'
+    CLDR_VERSION    = '26'
```
Comment (KL-7) [inline lib/twitter_cldr/resources/download.rb]: We have `CLDR_VERSION = '26.0'` in `lib/twitter_cldr.rb`. Can we combine these two somehow? I'd also suggest moving version constants for Unicode and ICU into `lib/twitter_cldr.rb` too.

File: lib/twitter_cldr/resources/download.rb [hunk 2]
```diff
@@ -51,6 +56,16 @@ def download_icu4j_if_necessary(path, url = ICU4J_URL)
...
       end
 
+      def download_unicode_data_if_necessary(path, url)
+        download_if_necessary(path, join_urls(UNICODE_URL, url))
+      end
+
+      private
+
+      def join_urls(*urls)
+        urls.map { |u| u.sub(/[\\\/]$/, '') }.join('/')
```
Comment (KL-7) [inline lib/twitter_cldr/resources/download.rb]: I think you can use `u.chomp('/')` here to avoid this scary escaping.

**Comment by KL-7** [comment]
Few minor comments, LGTM otherwise :shipit:

File: lib/twitter_cldr/resources/download.rb (hunk 1 above)
Comment (camertron) [inline lib/twitter_cldr/resources/download.rb]: That's a great idea. I could also move them into version.rb, or perhaps create a new module for them.

File: lib/twitter_cldr/resources/download.rb (hunk 2 above)
Comment (camertron) [inline lib/twitter_cldr/resources/download.rb]: :fearful: escaping

File: lib/twitter_cldr/resources/download.rb (hunk 1 above)
Comment (KL-7) [inline lib/twitter_cldr/resources/download.rb]: I'd just leave them at the top of the root `TwitterCldr` module to make them easy to find.

**Pull request merged**